README.md

```diff
@@ -1,2 +1,17 @@
-# docs
-Geo Engine Documentation
+# Geo Engine Docs
+
+A central place for all Geo Engine user documentation.
```

Comment:
```suggestion
A central place for all Geo Engine user documentations.
```
either plural or replace the "all" with "the".

Reply:
ccf6de3